Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/MoveToTarget.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System.Collections;
+using System.Net.NetworkInformation;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This class provides functionality to move the hand hint from a tracking position to a target position over time.
+    /// </summary>
+    public class MoveToTarget : MonoBehaviour
+    {
+        [Tooltip("Object to track.")]
+        [SerializeField]
+        private GameObject trackingObject = null;
+
+        /// <summary>
+        /// Object to track.
+        /// </summary>
+        public GameObject TrackingObject
+        {
+            get
+            {
+                return trackingObject;
+            }
+            protected set
```

Comment:
any reason this is protected? If it's protected, other classes will not be able to access this.

Reply:
Ah, I was just following the guidelines. I'll make them not protected